SignalMessaging/attachments/ApproveContactShareViewController.swift

```diff
@@ -55,6 +55,41 @@ class ContactShareFieldBase<ContactFieldType: OWSContactField>: NSObject, Contac
 
 // MARK: -
 
+// Stub class so that avatars conform to OWSContactField.
+class OWSContactAvatar: NSObject, OWSContactField {
```

Comment:
Maybe this was a bad idea, but I was trying to maintain the pattern despite some friction.